bluesky_kafka/kafka.py

```diff
@@ -7,13 +7,27 @@
 
 
 def delivery_report(err, msg):
-    """ Called once for each message produced to indicate delivery result.
-        Triggered by poll() or flush(). """
+    """
+    Called once for each message produced to indicate delivery result.
+    Triggered by poll() or flush().
+
+    Parameters
+    ----------
+    err
+    msg
+
+    Returns
+    -------
+
+    """
     if err is not None:
-        print('Message delivery failed: {}'.format(err))
+        print("Message delivery failed: {}".format(err))
```

Comment:
Is it worth using `logging` here as we do elsewhere?

Reply:
yes

Reply:
done